usmqe_tests/api/tendrl/others/conftest.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+
+@pytest.fixture(
+    params=[{
+        "name": "Tom Hardy",
+        "username": "thardy",
+        "email": "thardy@tendrl.org",
+        "role": "admin",
+        "password": "pass1234",
+        "password_confirmation": "pass1234"}])
+def valid_user_import(request):
+    """Generate valid data that can be imported into tendrl as a new user.
```

Comment:
I guess that the idea here is to try how  parametrization would work in this case. Moreover this way, we could easily add another user here.